tests/sanity/ignore-2.10.txt

```diff
@@ -48,8 +48,6 @@ plugins/modules/system/iptables_state.py validate-modules:undocumented-parameter
 plugins/modules/system/osx_defaults.py validate-modules:parameter-state-invalid-choice
 plugins/modules/system/parted.py validate-modules:parameter-state-invalid-choice
 plugins/modules/system/puppet.py use-argspec-type-path
```

Comment:
You need to add
```suggestion
plugins/modules/system/puppet.py validate-modules:parameter-invalid  # invalid alias - removed in 7.0.0
```
here and in the other ignore.txt files. (See here why it doesn't complain about composer: https://github.com/ansible-collections/community.general/blob/053b8a064db08b3c428b37a86b630f8e7b62b15d/tests/sanity/ignore-2.10.txt#L26)

Reply:
Ha right, ty!

Reply:
I put back `use-argspec-type-path` through another commit.